### PR TITLE
Document product header value

### PR DESCRIPTION
**Pull request by Haacked**
Fixed a parameter name that was misleading and documented the methods and properties.

### CODE REVIEW

**Comment by alfhenrik** [comment]
LGTM :smiley:

File: Octokit/Http/ProductHeaderValue.cs [hunk 1]
```diff
@@ -19,11 +36,17 @@ public ProductHeaderValue(string name, string value)
...
+        /// <summary>
+        /// The name of the product that's using Octokit
+        /// </summary>
         public string Name
         {
             get { return _productHeaderValue.Name; }
         }
 
+        /// <summary>
+        /// Gets the version of the product.
```
Comment (shiftkey) [inline Octokit/Http/ProductHeaderValue.cs]: :lipstick: `The version of the product` (i.e. drop the `Gets`)?

**Comment by shiftkey** [comment]
One little nit, everything else seems great!

File: Octokit/Http/ProductHeaderValue.cs (hunk 1 above)
Comment (Haacked) [inline Octokit/Http/ProductHeaderValue.cs]: Ha! Ghost Doc. :stuck_out_tongue:

**Comment by Haacked** [comment]
:kimono:

**Pull request merged**